src/app/design/downloads/index.html

```diff
@@ -0,0 +1,12 @@
+<stache 
+  layout="container"
+  pageTitle="Downloads"
```

Comment:
What do you think of "Templates" as a title? I assume the emphasis in the title and navigation should be that the section provides templates, not that it includes something downloadable.